src/constants/cookieNameKey.js

```diff
@@ -12,3 +12,4 @@ governing permissions and limitations under the License.
 
 export const IDENTITY = "identity";
 export const CONSENT = "consent";
+export const AT_QA_MODE = "at_qa_mode";
```

Comment:
Since this is a target-specific cookie name, I wonder if this would be better placed within the Personalization component?

Reply:
I think we are following DRY principle here. all the cookies at same place, less code & easy to maintain.

Reply:
I think we are following DRY principle here. all the cookies at same place, less code & easy to maintain.